### ISSUE TITLE
ainput doesn't work with uvloop

### ISSUE THREAD

**Issue by Busimus** [vxgmichel/aioconsole#17]
When using uvloop, I get this exception whenever I run `ainput`:

```
Traceback (most recent call last):
  File "/home/user/client.py", line 88, in client
    msg = await ainput("Message: ")
  File "/usr/lib/python3.6/site-packages/aioconsole-0.1.7.dev0-py3.6.egg/aioconsole/stream.py", line 157, in ainput
    reader, writer = yield from get_standard_streams(loop=loop)
  File "/usr/lib/python3.6/site-packages/aioconsole-0.1.7.dev0-py3.6.egg/aioconsole/stream.py", line 145, in get_standard_streams
    cache[key] = yield from connection
  File "/usr/lib/python3.6/site-packages/aioconsole-0.1.7.dev0-py3.6.egg/aioconsole/stream.py", line 133, in create_standard_streams
    reader, writer = yield from future
  File "/usr/lib/python3.6/site-packages/aioconsole-0.1.7.dev0-py3.6.egg/aioconsole/stream.py", line 117, in open_pipe_connection
    loop.remove_reader(transport.get_extra_info('pipe').fileno())
  File "uvloop/loop.pyx", line 2088, in uvloop.loop.Loop.remove_reader
  File "uvloop/loop.pyx", line 681, in uvloop.loop.Loop._remove_reader
  File "uvloop/loop.pyx", line 649, in uvloop.loop.Loop._ensure_fd_no_transport
RuntimeError: File descriptor 1 is used by transport <WriteUnixTransport closed=False reading=True 0x7f177a34d3d8>
```

Happens on both the release and dev versions. 
uvloop version is 0.9.1.

**Comment by vxgmichel** [comment]
Thanks for the report! For the record, this issue does not occur with uvloop 0.8.1.

**Comment by vxgmichel** [comment]
Fixed in d8dc5b0, will become visible in 0.1.7. Thanks again.

**Comment by vxgmichel** [comment]
[v0.1.7](https://github.com/vxgmichel/aioconsole/releases/tag/v0.1.7) is out!
